### PR TITLE
added files for the alpha stat geog local gov district nir register

### PR DESCRIPTION
**Pull request by Jamie-Atkinson**
files for the alpha version

### CODE REVIEW

File: data/alpha/field/local-authority-nir.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+text: "A local authority in Northern Ireland."
+field: "local-authority-nir"
+phase: "alpha"
+datatype: "string"
+register: ""
```
Comment (nacnudus) [inline data/alpha/field/local-authority-nir.yaml]: Doesn't this have to be `register: "local-authority-nir", and the local-authority-nir register cloned into alpha?

Comment (Jamie-Atkinson) [reply data/alpha/field/local-authority-nir.yaml]: for the links to work, yes. but I wasn't going to get them working. it's a non-contentious register.

Comment (nacnudus) [reply data/alpha/field/local-authority-nir.yaml]: @michaelabenyohai @elliecraven what's our current process?

Comment (elliecraven) [reply data/alpha/field/local-authority-nir.yaml]: Sounds good to me @michaelabenyohai happy to go with option 2 as our standard practice in these cases (unless we're later inundated with support queries that this is causing problems for users)

Comment (michaelabenyohai) [reply data/alpha/field/local-authority-nir.yaml]: Thanks @elliecraven . @Jamie-Atkinson that means you will need to populate `register: "local-authority-nir` here

**Pull request merged**